newsroom/agenda/views.py

```diff
@@ -15,7 +15,7 @@
 from newsroom.products.products import get_products_by_company
 from newsroom.template_filters import is_admin_or_internal, is_admin
 from newsroom.topics import get_company_folders, get_user_folders, get_user_topics
-from newsroom.navigations.navigations import get_navigations
+from newsroom.navigations.utils import get_navigations
```

Comment:
Would it be possible to add the following to the `__all__` attribute in `newsroom.navigations.__init__.py`:
- get_navigations
- get_navigation_by_company
- get_navigations_as_list

This way we can define what the `newsroom.navigations` module exposes, and not need to import deeper into that module

`from newsroom.navigations.utils import get_navigations`
would then become
`from newsroom.navigations import get_navigations`